### PR TITLE
First stab at creating the Dockerfile for this

### PR DESCRIPTION
**Pull request by BenWhitehead**

### CODE REVIEW

**Comment by BenWhitehead** [comment]
@timfallmk Here is a dockerfile to get going with this.

**Comment by timfallmk** [comment]
@BenWhitehead Damn, you beat me to it. Forgot to push my commits :).

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+RUN mkdir /tmp/dcos-cli
+RUN apt-get update && apt-get install -y wget curl python-pip less tmux
+
+RUN pip install virtualenv httpie
+
+ENV DCOS_CONFIG=/root/.dcos/dcos.toml
+
+ADD https://s3.amazonaws.com/downloads.mesosphere.io/dcos-cli/install.sh /tmp/
+RUN /bin/bash /tmp/install.sh /tmp/dcos-cli http://localhost || true
```
Comment (timfallmk) [inline Dockerfile:11]: @BenWhitehead Do you want to pull the config from the cluster URL?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+RUN wget -qO- https://downloads.mesosphere.io/java/jre-8u51-linux-x64.tar.gz | tar xvz -C /opt/
```
Comment (timfallmk) [inline Dockerfile:13]: Install java from upstream packages?

File: Dockerfile (hunk 1 above)
Comment (BenWhitehead) [inline Dockerfile:11]: The way I usually do it is to set the url after the fact, this value is valid for script install time.

File: Dockerfile (hunk 2 above)
Comment (BenWhitehead) [inline Dockerfile:13]: We'll probably need to change to that. It really shouldn't have to be there, but it's needed by the kafka cli.

**Comment by BenWhitehead** [comment]
obsolete

**Pull request closed without merging**